### ISSUE TITLE
Cut a new release?

### ISSUE THREAD

**Issue by rndmh3ro** [BigBoot/AutoKuma#82]
Thanks for your great tool - I'm using it to automate monitor-creation in kubernetes.

Is it possible that you create a new release so I can use the feature from https://github.com/BigBoot/AutoKuma/commit/e936f056fe6c769f3b18dbd3e6e88217dd59b83a, without having to use cargo?

That'd be awesome!

**Comment by johntdyer** [comment]
Quite a few fixes since this request and we're still on 0.8.0... Any plans to cut a new build ?

**Comment by BigBoot** [comment]
Hmm yes I wanted to wait for uptime-kuma V2 release but I didn't expect it to take this long. I guess a new release is a good idea. 

That said uptime kuma v1 is still supported (and even default) on master, there are currently 2 builds:  
`ghcr.io/bigboot/autokuma:master` for v1
`ghcr.io/bigboot/autokuma:uptime-kuma-v2-master` for v2 
the same prefixed versions are also available for all of the commit-based tags too.

**Comment by BigBoot** [comment]
Yes, `ghcr.io/bigboot/autokuma:master` is the current master branch (i.e. 2f4511c5c8bbbcc27a47e1e1a2436967a0be211f as of right now) built with uptime kuma v1 support.
